build.gradle

```diff
@@ -60,33 +60,18 @@ allprojects {
                 username geArtifactoryReader
                 password geArtifactoryReaderPassword
             }
-            url 'https://devcloud.swcoe.ge.com/artifactory/PREDIX'
-        }
-        maven {
-            credentials {
-                username geArtifactoryReader
-                password geArtifactoryReaderPassword
-            }
-            url 'https://devcloud.swcoe.ge.com/artifactory/PREDIX-SNAPSHOT'
+            url 'https://artifactory.build.ge.com/predix-virtual'
         }
         // Repositories required to download jwt-bearer-key-provider
         maven {
             credentials {
                 username geArtifactoryReader
                 password geArtifactoryReaderPassword
             }
-            url 'https://devcloud.swcoe.ge.com/artifactory/MAAXA-MVN'
-        }
-        maven {
-            credentials {
-                username geArtifactoryReader
-                password geArtifactoryReaderPassword
-            }
-            url 'https://devcloud.swcoe.ge.com/artifactory/MAAXA-MVN-SNAPSHOT'
-        }
-        maven {
-            url 'https://build.shibboleth.net/nexus/content/repositories/releases/'
+            url 'https://artifactory.build.ge.com/artifactory/MAAXA'
         }
+        maven { url 'https://artifacts.alfresco.com/nexus/content/repositories/public/' }
+        maven { url 'https://build.shibboleth.net/nexus/content/repositories/releases/' }
```

Comment:
"build.shibboleth.net" is not reachable. It was removed in this commit https://github.com/GESoftware-CF/uaa/commit/74c1ec27e004ce4d3822c99b3637204eac00c527. Should we also include the commit in this PR?

Reply:
Good catch.  This branch should never be built from Jenkins. However, to be consistent and make merging of this branch into predix_extensions in the future easier, it makes sense to pick up this commit as well.

Reply:
Thanks @ylongva & @irinaepshteyn, I cherry picked that commit as well in this PR